app/utils/image_cache_manager.js

```diff
@@ -27,7 +27,7 @@ export default class ImageCacheManager {
             listener(path);
         } else {
             addListener(uri, listener);
-            if (uri.startsWith('file://')) {
+            if (!uri.startsWith('http')) {
```

Comment:
What if someone tries to link to a file some other way, like an image on an ftp server? Maybe we want to check if the link doesn't start with any protocol like
```javascript
if (!(/^[^:/]:\/\//).test(uri)) {}
```

Reply:
yeah maybe, but we need to make sure the protocol does not include `file://` or `content://` can you come up with a regex for that?

Reply:
Could we just do `uri.startsWith('file://') || uri.startsWith('content://')` if it's just those two that we're looking out for?

Reply:
hmm, no.. there are also times when it just starts with a `/` at least on iOS

Reply:
Okay, this should be fine then. It's definitely a small change someone is using a link to an image on an ftp server, but that's small enough that we can fix it if it does become an issue